src/avt/Filters/avtGhostZoneFilter.C

```diff
@@ -318,6 +320,7 @@ avtGhostZoneFilter::ExecuteData(avtDataRepresentation *in_dr)
     vtkDataSetRemoveGhostCells *filter = vtkDataSetRemoveGhostCells::New();
     filter->SetGhostNodeTypesToRemove(ghostNodeTypesToRemove);
     filter->SetGhostZoneTypesToRemove(ghostZoneTypesToRemove);
+    filter->SetforceConfirmRegion(confirmRegion);
```

Comment:
Minor nit: 'SetforceConfirmRegion' would read better as 'SetForceConfirmRegion'.  I understand use of 'vtkSetMacro(confirmForceRegion, bool);' in vtkDataSetRemoveGhostCells is the cause. Suggest changing to 'ConfirmForceRegion' in the vtk class.

Reply:
@biagas ...I thought it was odd that the class variables were capitalized, now it makes sense...will change.